vocabularies/Org.OData.Temporal.V1.xml

```diff
@@ -95,11 +95,22 @@
       </ComplexType>
 
       <ComplexType Name="CollectionKindSnapshot" BaseType="Temporal.CollectionKindType">
-        <Annotation Term="Core.Description" String="Each OData entity corresponds to a temporal object and by default only shows data of the current time slice" />
+        <Annotation Term="Core.Description" String="Each OData entity represents a snapshot of data at a point in application time that is the same for all entities in the collection" />
+        <Annotation Term="Core.LongDescription">
+          <String>The point in application time is defined by the `$at` clause of a request; default is the current point in time.</String>
+        </Annotation>
       </ComplexType>
 
       <ComplexType Name="CollectionKindTimeSlice" BaseType="Temporal.CollectionKindType">
-        <Annotation Term="Core.Description" String="Each OData entity corresponds to a time slice and all time slices of a temporal object are part of the collection" />
+        <Annotation Term="Core.Description" String="Each OData entity represents data during an interval of application time; it is called a *time slice*" />
+        <Annotation Term="Core.LongDescription">
+          <String>Entities in a temporal collection of this kind are said to *overlap*
+          if the intervals defined by their `TimeSliceStart` and `TimeSliceEnd` properties overlap
+          and their other, *non-temporal*, key properties are identical.

```

Comment:
Are `TimeSliceStart` and `TimeSliceEnd` always key properties (in the OData sense), or can they also be part of a "logical key"? That would complicate the definition of "overlapping".

Reply:
They can be non-key. "Overlapping" should only be used for the time period, not for the "object key"

Reply:
> "Overlapping" should only be used for the time period, not for the "object key"

A time-slice collection may contain overlapping intervals if the object keys are different. That's why I define "overlapping entities" by looking also at the object key. But since the object key can also be non-key, we must make it explicit (see https://github.com/oasis-tcs/odata-vocabularies/pull/128#discussion_r650726899).